rollup.config.js

```diff
@@ -0,0 +1,35 @@
+import { readFileSync } from 'fs';
+import buble from 'rollup-plugin-buble';
+import uglify from 'rollup-plugin-uglify';
+import commonjs from 'rollup-plugin-commonjs';
+import replace from 'rollup-plugin-replace';
+
+const pkg = JSON.parse( readFileSync( 'package.json', 'utf-8' ) );
+
+const isProduction = process.env.NODE_ENV === 'production';
+
+const banner = readFileSync( 'banner.js', 'utf-8' )
+	.replace( '${name}', pkg.name )
+  .replace( '${version}', pkg.version )
+  .replace( '${author}', pkg.author )
+  .replace( '${homepage}', pkg.homepage )
+
+export default {
+  input: `./lib/index.js`,
+  plugins: [
+    isProduction ? uglify({}) : {},
```

Comment:
This might be conditional - I'd imagine if you are using this node-only you'd like to avoid the uglification.